### DISCUSSION TITLE
MutableEthernetPacket definition is missing

### DISCUSSION THREAD

**Discussion by yuchowchen** [libpnet/libpnet#528] [General]
use pnet::packet::ethernet::{EtherTypes, EthernetPacket, MutableEthernetPacket};

in version pnet ="0.28.0",A lot of files use MutableEthernetPacket structure, but in the source file, the definition is missing. Would you please check it ?

**Comment by infosechoudini** [comment]
Hey! What do you mean it's missing in the source? Mutable and Immutable packets are defined by the traits in macro_support
